web/js/components/map/ol-vector-interactions.js

```diff
@@ -0,0 +1,113 @@
+import {
```

Comment:
👍   I really like this idea of combining a specific _kind_ of map interactions into their own component.  I'm going to try to convert the measure tool to a React component like this one.

Reply:
sounds great. It would be good to normalize as much of that `map/` code as possible